src/config.py

```diff
@@ -174,7 +174,7 @@ def load_data():
             "info_screenshots": bool(int(Config._select_val("general", "info_screenshots"))),
             "loot_screenshots": bool(int(Config._select_val("general", "loot_screenshots"))),
             "d2r_path": Config._select_val("general", "d2r_path"),
-            "restart_d2r_when_stuck": bool(int(Config._select_val("general", "restart_d2r_when_stuck"))),
```

Comment:
Why you dont like trailing `,` :( 
Its nice when a new param is added there will be just one line change instead of two and less risk of merge conflicts of others. It's not critical and probably doesn't happen very often that it really helps, still doesn't hurt to do.

Reply:
It's not a thing in the language I've been working for over a decade :). But I see your point